package.json

```diff
@@ -47,15 +47,16 @@
     "@types/koa": "^2.0.46",
     "@types/koa-bodyparser": "^5.0.1",
     "@types/koa-compress": "^2.0.8",
+    "@types/koa-logger": "^3.1.1",
     "@types/koa-route": "^3.2.4",
     "@types/koa-send": "^4.1.1",
     "@types/koa-static": "^4.0.0",
     "@types/puppeteer": "^1.6.0",
     "@types/supertest": "^2.0.4",
-    "@types/koa-logger": "^3.1.1",
     "ava": "1.0.0-beta.6",
     "nodemon": "^1.18.3",
     "supertest": "^3.0.0",
+    "ts-node": "^8.0.2",
```

Comment:
Why do we need this additional dependency?

Reply:
@AVGP I have removed that dependency as it is no longer needed. Also I have reverted the package-lock.json file to the one from master. See my comment below for details. I will let you mark this review comment resolved.